README.md

```diff
@@ -13,7 +13,7 @@ Process XDR types based on the [RFC4506](https://www.ietf.org/rfc/rfc4506.txt).
 ```elixir
 def deps do
   [
-    {:elixir_xdr, "~> 0.1.0"}
+    {:elixir_xdr, "~> 0.1.3"}
```

Comment:
@marioyustikommit 0.1.4 is the next version. Could you please also update the readme file (only if it is required)

Reply:
Yes, I updated the README and the Guides of Void and Union.